src/vector/platform/index.js

```diff
@@ -16,8 +16,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+import ElectronPlatform from './ElectronPlatform';
 import WebPlatform from './WebPlatform';
 
-let Platform = WebPlatform;
+let Platform = null;
+
+if (window && window.process && window.process && window.process.type === 'renderer') {
```

Comment:
can we make this happen at build time (and make the Web/ElectronPlatform import conditional) so as to avoid shipping electron in the web bundle (assuming that's what happens)?